homeassistant/components/blebox/cover.py

```diff
@@ -0,0 +1,133 @@
+"""BleBox cover entity."""
+
+import logging
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    DEVICE_CLASS_DOOR,
+    DEVICE_CLASS_SHUTTER,
+    STATE_CLOSED,
+    STATE_CLOSING,
+    STATE_OPEN,
+    STATE_OPENING,
+    SUPPORT_CLOSE,
+    SUPPORT_OPEN,
+    SUPPORT_SET_POSITION,
+    SUPPORT_STOP,
+    CoverDevice,
+)
+from homeassistant.exceptions import PlatformNotReady
+
+from . import CommonEntity, async_add_blebox
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add, discovery_info=None):
```

Comment:
You should be able to remove this since its setup via the config entry and not a platform

Reply:
Removed, thanks!